### PR TITLE
Trancscript10

### PR DESCRIPTION
**Pull request by deeayeen**
Agenda headers included in transcript. Happy to do future transcripts for a slightly higher bounty 👍

### CODE REVIEW

**Comment by oskarth** [comment]
Thanks @deeayeen! I'll have a look at this on Monday, hope that's OK. 

@PombeirP @hesterbruikman would you mind having a look too? Couldn't request you to review for some reason.

**Comment by deeayeen** [comment]
@hesterbruikman @oskarth
Thanks, fixed

File: notes/transcript10.md [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+
+# Core Network Incentives
+11:00 - 12:00
+
+Oskar: Cool, so network incentives?
+
+Status: Yeah we are continuing our research like Andrea did research on one of the ideas of incentive of the current infrastructure meaning whisper and mail servers. For the next week we wanna also take a look like taken into account that's there's also a protocol research happening so we want something more broad and applicable and future proof. So people want to have some research on something different and not like whisper side, just come up with something that's more generic and durable so that's essentially it I hope we'll share the documents this week, the drafts of research.
```
Comment (oskarth) [inline notes/transcript10.md]: Status should be Igor here (and elsewhere)

File: notes/transcript10.md [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+
+# Protocal Update
+14:00 - 15:00
+
+Oskar: Cool thank you. I'll do the protocol update. So mainly two things have been done. So one is workshop preparation. We're gonna meet just before Fossum in Brussels end of January, we'll be fine to seven people from sellers. So working on agenda and making as useful to as many people as possible. The other thing is proof of concept for a sort of data sync layer. I've written a post about this before but the idea is that you encapsulate an in between layer that doesn't concern itself with a message per say but instead of just syncing a piece of data and that allows us to build things on top of it. Sort of public chat and group chat and so on, but below it you can swap whisper out and still be able to sync data in a peer-to-peer way. In this proof concept there is a proto-buff specification that is wrong in some places but gets the jist across.
```
Comment (oskarth) [inline notes/transcript10.md]: Fossum -> Fossdem
sellers -> Status

proto-buff -> protobuf

just -> gist

File: notes/transcript10.md [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+
+# Dao
+15:00 - 16:00
+
+Oskar: As well as a sort of basic simulation in Python that has some unreliability assumptions and also deals with mobile node having slightly different characteristics than having something like a desktop note where a mobile node is mostly offline but sometimes goes online. Then sort of for a one-to-one chat and seeing how the system behaves, so I did a short write-up, I'll link to it in the notes, yeah please ask questions or specific things you want me to try to answer with this concept. Please ping me as well. This week we will continue to work from the workshop and looking at what original direction the future will constantly bring. dao?
```
Comment (oskarth) [inline notes/transcript10.md]: "looking at what original direction the future will constantly bring" huh did I really say that :D doesn't make much sense

**Comment by oskarth** [comment]
Oh, would you mind adding this as a link 'transcript' from README as well? Under the 10th issue in the table

File: notes/transcript10.md (hunk 2 above)
Comment (deeayeen) [reply notes/transcript10.md]: Fixed

File: notes/transcript10.md (hunk 1 above)
Comment (deeayeen) [reply notes/transcript10.md]: Fixed on 65

**Comment by deeayeen** [comment]
> Oh, would you mind adding this as a link 'transcript' from README as well? Under the 10th issue in the table

@oskarth 

Of course! PR'ed all the fixes and added to readme.

All in all it took me about two and a half hours. I also learned the concepts along the way, so future transcriptions will be more accurate.

**Comment by oskarth** [comment]
Great, cheers @deeayeen! Feel free to join us in #status-core-devs if you want as well :)

That was slightly more than expected, so I added two more issues with "size-s" (previous was size-xs), pending funding. cc @StatusSceptre

**Pull request merged**